File: packages/expo-modules-core/ios/Fetch/NetworkFetchExceptions.swift
// Copyright 2015-present 650 Industries. All rights reserved.

internal class NetworkFetchURLSessionLostException: Exception {
  override var reason: String {
    "The url session has been lost"
  }
}

internal class NetworkFetchUnknownException: Exception {
  override var reason: String {
    "Unknown error"
  }
}

internal class NetworkFetchRequestCancelledException: Exception {
  override var reason: String {
    "Cancelled request"
  }
}

File: packages/expo/android/src/main/java/expo/modules/networkfetch/NetworkFetchExceptions.kt
// Copyright 2015-present 650 Industries. All rights reserved.

package expo.modules.networkfetch

import expo.modules.kotlin.exception.CodedException

internal class NetworkFetchUnknownException :
  CodedException("Unknown error")

internal class NetworkFetchRequestCancelledException :
  CodedException("Cancelled request")
